packages/features/bookings/lib/getBookingResponsesSchema.ts

```diff
@@ -246,6 +247,17 @@ function preprocess<T extends z.ZodType>({
           continue;
         }
 
+        if (bookingField.type === "url") {
+          // URL schema to validate if the input is a valid URL
+          if (!urlSchema.safeParse(value).success) {
+            ctx.addIssue({
+              code: z.ZodIssueCode.custom,
+              message: m("url_validation_error"),
+            });
+          }
+          continue;
+        }
```

Comment:
https://github.com/calcom/cal.com/blob/f23a683bafca1cfc0022ef8eaa47cb797c24cd3b/packages/features/form-builder/schema.ts#L320 It would be the right place to add the `superRefine` for `url` type field in the same way we have done for 'name' type field.

We want to avoid increase this file's size and make all refinements testable.
After that it would make sense to add a simple test for url type field schema validation

Reply:
Done